src/plugins/intel_gna/README.md

```diff
@@ -0,0 +1,69 @@
+# GNA Plugin Developer Documentation
+
+Welcome to the GNA Plugin Developer Documentation. This documentation helps deeper understand the GNA Plugin architecture and gives detailed information on the concepts and ideas used inside.
+
+The GNA plugin provides a way to run inference on Intel® GNA, as well as in the software execution mode on CPU.
+
+## Get Started
+ * [Build](./build.md)
+ * How to:
+    * [GNA configuration](https://docs.openvino.ai/latest/openvino_docs_install_guides_configurations_for_intel_gna.html)
+    * [Add new transformation](#todo)
+ * [debug capabilities](./docs/debug_capabilities.md)
+ * [Code Style](./docs/code_style.md)
+
+## Key contacts
+* Denis Orlov <denis.orlov@intel.com>
+* Mikhail Ryzhov <mikhail.ryzhov@intel.com>
+* Evgeny Kotov <evgeny.kotov@intel.com>
+* Szymon Jakub Irzabek <szymon.jakub.irzabek@intel.com>
+* Nadezhda Ageeva <nadezhda.ageeva@intel.com>
+
```

Comment:
Just wonder if make sense to add following chapters e.g.:

## Glossary
| Name | Description|
|------|------------|
| GNA | Gaussian & Neural Accelerator |
| dnn | |
| CNN | |

## Delivered API
GNA plugin does not define API. It is dynamically loaded. Interface delivered by plugin is defined in.
[Openvion Dev API](https://github.com/marcinkusm/openvino/tree/master/src/inference/dev_api/cpp_interfaces/interface)

## Dependencies

### Build time dependencies
TODO
| Dependency | Purpose | comment |
|------------|---------|---------|
| ngraph | | |
| GNA library | | |

### Runtime Dependencies
TODO
| Dependency | Purpose | comment |
|------------|---------|---------|
| ngraph | | |
| GNA library | | |